src/main/java/time/TimeInCapitals/showtime/ShowTime.java

```diff
@@ -13,24 +13,10 @@
 
 @Controller
 public class ShowTime {
-
-	@GetMapping("/")
-	@ResponseBody
-	public String getTimeDate() {
-
-		DateTimeFormatter timeFormat = DateTimeFormatter.ofPattern("H:m:s");
-		DateTimeFormatter dateFormat = DateTimeFormatter.ofPattern("d MM y");
-		LocalDateTime localDateTime = LocalDateTime.now();
-
-		String localTime = localDateTime.format(timeFormat);
-		String localDate = localDateTime.format(dateFormat);
-
-		return "Local Time: " + localTime + "\nLocal date: " + localDate;
-	}
-
-	@GetMapping({ "/gettime/{td}", "/gettime" })
+	
+	@GetMapping({"/", "/localtime", "/localtime/{timeOrDate}" })
```

Comment:
`GetMapping("/localtime/{timeOrDate}")`

Reply:
Czyli dla każdego mapowania zrobić osobną metodę?
Przykładowo /localtime bez parametru, albo samo "/"?

Chciałbym też coś wyświetlić jeśli zostanie wpisane samo /localtime, albo "/"

Reply:
Ok, ale po co? Lepiej zdecydować się na jednoznaczny endpoint, nigdy nie widziałem aby ktoś robił jeden endpoint mający 3 różne adresy

Ewentualnie można zrobić przekierowanie, ale to jest inna bajka